gulpfile.js

```diff
@@ -64,6 +55,11 @@ gulp.task('css', () => {
     .pipe(gulp.dest('dist/public/stylesheets/'));
 });
 
+gulp.task('vendor-assets', () => {
+  gulp.src([
+    'node_modules/govuk-elements-sass/node_modules/govuk_frontend_toolkit/images/*.png',
+  ]).pipe(gulp.dest('vendor/govuk_frontend_toolkit/assets/images'));
```

Comment:
As the vendor directory is checked in, I think we should output the vendor-assets to dist/